ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -87,13 +87,8 @@ compute:
   replicas: ${workers}
 platform:
   baremetal:
-    libvirtURI: >-
```

Comment:
This is required to create the bootstrap VM in the target provisioner

Reply:
Yes, sure. it's just for testing. will revert it soon